### PR TITLE
Ss post beta profile updates

### PR DESCRIPTION
**Pull request by snsebro**
What
Fix bug where user profile hangs after trying to create a new user
Fix bug with updating user profile

How
Updated `createUser() ` when called in the `CreateUser` component to be triggered by a `useEffect` whenever the page status is changed.

Changed user identites being submitted when updating a profile to be an array of nested objects 
`identities = [{ name: identity }, { name: identity }, ...]`

### CODE REVIEW

File: src/api/wrappedFetch.js [hunk 1]
```diff
@@ -20,6 +20,7 @@ const wrappedFetch = async (url, opts) => {
     error.status = 404;
     throw error;
   } else {
+    console.log(result);
```
Comment (deepakvenkat) [inline src/api/wrappedFetch.js:23]: Let's remove this?

File: src/hooks/useAuthenticatedUser.js [hunk 2]
```diff
@@ -41,6 +41,7 @@ const useAuthenticatedUser = () => {
           token,
         });
         profile = await getUserProfile(registeredUser.data.id, token);
+        console.log(profile.data.user);
```
Comment (deepakvenkat) [inline src/hooks/useAuthenticatedUser.js:44]: Also this

**Review by deepakvenkat — APPROVED** [review]
Looks good to merge after removing the console logs.

File: src/hooks/useAuthenticatedUser.js (hunk 2 above)
Comment (snsebro) [reply src/hooks/useAuthenticatedUser.js:44]: Ah good catch, udpated!

File: src/api/wrappedFetch.js (hunk 1 above)
Comment (snsebro) [reply src/api/wrappedFetch.js:23]: updated!

**Pull request merged**